plugins/Zeroname/updater/zeroname_updater.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function
```

Comment:
Do we still need to support Python 2?  If so, this line can not be deleted or Python 2 will treat the comma on line 29 as a syntax error.

Reply:
I believe ZeroNet doesn't support py2 anymore.